### ISSUE TITLE
Questions about the validation set

### ISSUE THREAD

**Issue by GA-17a** [KaiyangZhou/ssdg-benchmark#8]
Hi,
Did you use the validation set in the model training? Considering that in PACS and Office-Home datasets, the number of the validation set is much larger than the number of the training set under semi-supervised DG setting, I think it is unreasonable to use such large validation sets to select hyperparameters or the final models because it is inconsistent with reality. I observe that you keep validation sets in the data splits, what are they used for?

**Comment by KaiyangZhou** [comment]
no, they aren't used in the ssdg setting

we followed the original data split so the results are comparable to published work (to have a sense of what ssdg methods could acieve)

**Comment by GA-17a** [comment]
Did you select hyperparameters based on the accuracy of test set?

**Comment by KaiyangZhou** [comment]
no, there is no algorithmic hyper-param to tune and the train params simply follow commonly used ones

**Comment by GA-17a** [comment]
Thanks for your reply

**Comment by KaiyangZhou** [comment]
no problem :)

feel free to open new issues
